app/controllers/orders_controller.rb

```diff
@@ -14,7 +14,7 @@ def create
         total_price: total_price
       )
       add_items_to_order
-      flash[:order_success] = {color: 'green', message: "Order was successfully placed}"}
+      flash[:order_success] = {color: 'green', message: "Order was successfully placed"}
```

Comment:
Line is too long. [88/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.